doc/src/manual/control-flow.md

```diff
@@ -842,52 +842,55 @@ them. The consumer cannot simply call a producer function to get a value, becaus
 may have more values to generate and so might not yet be ready to return. With tasks, the producer
 and consumer can both run as long as they need to, passing values back and forth as necessary.
 
-Julia provides the functions [`produce()`](@ref) and [`consume()`](@ref) for solving this problem.
-A producer is a function that calls [`produce()`](@ref) on each value it needs to produce:
+Julia provides a [`Channel`](@ref) mechanism for solving this problem.
+A [`Channel`](@ref) is a waitable FIFO queue which can have multiple tasks reading and writing to it.
```

Comment:
I can ask a dozen julia users in my lab tomorrow, which of them know what FIFO means, at most one is going to. We avoid acronyms in package names, we should avoid unexplained acronyms in documentation.